### PR TITLE
Added name to contributors.md

### PR DESCRIPTION
**Pull request by jayeryden**
This should add just my name to the contributors file

### CODE REVIEW

File: docs/portfolios.md [hunk 1]
```diff
@@ -3,7 +3,3 @@
 💕 [blwatkins](https://blwatkins.github.io/)
 
 ***[Makayla Alston](https://makayla-a.github.io/)***
-
-***[Tanner Shepherd](https://tzshepherd.github.io/)***
```
Comment (blwatkins) [inline docs/portfolios.md]: Changes in this file should not remove other students' entries.

**Review by blwatkins — CHANGES_REQUESTED** [review]
Please fix the merge conflicts in this file.  Check the "Files changed" tab after you create or your pull request to make sure that the changes match what you expect.

Changes in this pull request should not remove changes made by other students (see [docs/portfolios.md changes](https://github.com/blwatkins/CRCP3320-1247/pull/38/files#diff-9b065e039ede99aa365535fe1cd434f2d2df9688ea0088f273961beee0de9f37L6-L9)).

Comment Date: Friday, October 18.

File: docs/portfolios.md (hunk 1 above)
Comment (jayeryden) [reply docs/portfolios.md]: How do I fix this error? I didn't think I made these changes on my local machine and I have pulled so my branch is up to date. How should I resolve this? I see the changes in Files changed but I'm not sure how to fix the merge conflict and undo what I did.

**Comment by jayeryden** [comment]
Going to close this pull request and submit a replacement

**Pull request closed without merging**